permissions/permission_definitions.go

```diff
@@ -0,0 +1,699 @@
+				"CAN_RUN":    {isManagementPermission: false},
+				"CAN_EDIT":   {isManagementPermission: false},
+				"CAN_MANAGE": {isManagementPermission: true},
+			},
+			idRetriever:        PATH,
+			readAclCustomizers: []aclReadCustomizer{removeAdminPermissionsCustomizer},
+		},
+		{
+			field:             "authorization",
+			objectType:        "tokens",
+			requestObjectType: "authorization",
+			stateMatcher: func(id string) bool {
+				return id == "tokens"
+			},
+			idMatcher: func(id string) bool {
+				return id == "/authorization/tokens"
+			},
+			allowedPermissionLevels: map[string]permissionLevelOptions{
+				"CAN_USE":    {isManagementPermission: true},
+				"CAN_MANAGE": {isManagementPermission: true},
+			},
+			updateAclCustomizers: []aclUpdateCustomizer{
+				addAdminAclCustomizer(func(objectId string) bool {
+					return objectId == "/authorization/tokens"
+				}),
+			},
+		},
+		{
+			field:             "authorization",
+			objectType:        "passwords",
+			requestObjectType: "authorization",
+			stateMatcher: func(id string) bool {
+				return id == "passwords"
+			},
+			idMatcher: func(id string) bool {
+				return id == "/authorization/passwords"
+			},
+			allowedPermissionLevels: map[string]permissionLevelOptions{
+				"CAN_USE": {isManagementPermission: true},
+			},
+		},
+		{
+			field:             "sql_endpoint_id",
+			objectType:        "warehouses",
+			requestObjectType: "sql/warehouses",
+			allowedPermissionLevels: map[string]permissionLevelOptions{
+				"CAN_USE":     {isManagementPermission: false},
+				"CAN_MANAGE":  {isManagementPermission: true},
+				"CAN_MONITOR": {isManagementPermission: false},
+				"IS_OWNER":    {isManagementPermission: true},
+			},
+			updateAclCustomizers: []aclUpdateCustomizer{addCurrentUserAsManageCustomizer},
+			deleteAclCustomizers: []aclUpdateCustomizer{addCurrentUserAsManageCustomizer},
+			readAclCustomizers:   []aclReadCustomizer{removeAdminPermissionsCustomizer},
+			fetchObjectCreator: func(ctx context.Context, w *databricks.WorkspaceClient, objectID string) (string, error) {
+				warehouse, err := w.Warehouses.GetById(ctx, strings.ReplaceAll(objectID, "/sql/warehouses/", ""))
+				if err != nil {
+					return "", common.IgnoreNotFoundError(err)
+				}
+				return warehouse.CreatorName, nil
+			},
+		},
+		{
+			field:             "sql_dashboard_id",
+			objectType:        "dashboard",
+			requestObjectType: "dbsql-dashboards",
+			allowedPermissionLevels: map[string]permissionLevelOptions{
+				"CAN_EDIT":   {isManagementPermission: false},
+				"CAN_RUN":    {isManagementPermission: false},
+				"CAN_MANAGE": {isManagementPermission: true},
+				// This was originally called CAN_VIEW in the preview API, but was renamed to CAN_READ in the GA API.
+				"CAN_VIEW": {isManagementPermission: false},
+			},
+			idMatcher: func(id string) bool {
+				return strings.HasPrefix(id, "/dbsql-dashboards/") || strings.HasPrefix(id, "/sql/dashboards/")
```

Comment:
aren't we normalizing the ids to `dashboard` in the read function?

Reply:
Yes, but I think this is needed for `databricks_permissions` resources deployed on older versions of the provider.